docs/guides/runtime-options-overview.mdx

```diff
@@ -61,51 +61,51 @@ Scroll to see the full table.
-|                      | [layer_noise_learning](/api/qiskit-ibm-runtime/options-layer-noise-learning-options)     |      | `True`/`False`                | `True`               |      |
+| [max_execution_time](/api/qiskit-ibm-runtime/options-estimator-options#max_execution_time)   |                          |       | Integer number of seconds in the range  [1, 10800]    | `10800` (3 hours)   | max_execution_time   |
+| [resilience](/api/qiskit-ibm-runtime/options-resilience-options-v2)           |                          |      |                               |                      | resilience    |
+|                      | [layer_noise_learning](/api/qiskit-ibm-runtime/options-resilience-options-v2#layer_noise_learning)     |                         |                    |                            |      |
+|                      |                          | layer_pair_depths       |  list[int] of 2-10 values in the range [0, 200]                           | `(0, 1, 2, 4, 16, 32)`              |                      |
 |                      |                          | max_layers_to_learn     |  `None`/ integer >= 1                                                                      | `4`                    |                      |
-|                      |                          | shots_per_randomization |  integer >= 1                                                                         |  `128`                    |                      |
 |                      |                          | num_randomizations      |   integer >= 1                                                                            |  `32`                    |                      |
-|                      |                          | layer_pair_depths       |  list[int] of 2-10 values in the range [0, 200]                           | `(0, 1, 2, 4, 16, 32)`              |                      |
-|                      | [layer_noise_model](/api/qiskit-ibm-runtime/options-resilience-options-v2#layer_noise_model)       |        | `NoiseLearnerResult`/`Sequence[LayerError]`                      | `None`                      |                      |
+|                      |                          | shots_per_randomization |  integer >= 1                                                                         |  `128`                    |                      |
+|                      | layer_noise_model       |        | `NoiseLearnerResult`/`Sequence[LayerError]`                      | `None`                      |                      |
 |                      | measure_mitigation       |        | `True`/`False`                      | `True`                      |                      |
 |                      | [measure_noise_learning](/api/qiskit-ibm-runtime/options-measure-noise-learning-options)   | num_randomizations      |  integer >= 1  | `32`                      |                      |
```

Comment:
nit - the other sub option categories don't have values next to them, e.g. 

<img width="760" alt="image" src="https://github.com/user-attachments/assets/28aaba90-cead-41e1-a4df-3d3f68be6546" />

<img width="608" alt="image" src="https://github.com/user-attachments/assets/4703f5b6-d457-474c-a6da-d404499ee922" />


But it's different for `pec` and `zne`:

<img width="590" alt="image" src="https://github.com/user-attachments/assets/8199a72c-8d0b-4937-9489-6d5a940a11ea" />